src/data/user.ts

```
@@ -0,0 +1,10 @@
+import { db } from "@/db";
+
+export const getUserById = async (id: string) => {
+  try {
+    const user = await db.user.findUnique({ where: { id: id } });
+    return user;
+  } catch (e) {
+    return null;
```

why returning null? It is not better to throw an error or something?

---

maybe we could return some object with error info but i guess for now its nice and simple, you fetch user and either get user or null

---

thats right @bartosztrusinski